python/llm/src/bigdl/llm/transformers/loader.py

```diff
@@ -65,7 +65,7 @@ def load_model(
     # Load tokenizer
     tokenizer = tokenizer_cls.from_pretrained(model_path, trust_remote_code=True)
     model = model_cls.from_pretrained(model_path, **model_kwargs)
-    if not get_enable_ipex(low_bit):
+    if not get_enable_ipex_low_bit(low_bit):
```

Comment:
we can remove function `get_enable_ipex_low_bit`, just check by `get_enable_ipex()`. IPEX-optimized models always don't need `.eval()`